lxd/daemon.go

```diff
@@ -367,21 +367,21 @@ func (d *Daemon) SetupStorageDriver() error {
 	if lvmVgName != "" {
 		d.Storage, err = newStorage(d, storageTypeLvm)
 		if err != nil {
-			shared.LogInfof("Could not initialize storage type LVM: %s - falling back to dir", err)
+			shared.LogDebugf("Could not initialize storage type LVM: %s - falling back to dir", err)
```

Comment:
Feels to me like this one and the other similar storage errors should be LogErrorf.
The daemon will still work, but the configured storage backend won't be used.